sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityResult.java

```diff
@@ -58,23 +59,25 @@
     private SensitivityComputationStatus status;
     private final StateResult nStateResult = new StateResult();
     private final Map<String, StateResult> contingencyResults = new HashMap<>();
+    private final Map<String, StateResult> contingencyResultsAfterCra = new HashMap<>();
```

Comment:
Could we rename this attribute "postCraResults" and the previous one "postContingencyResults"?

Reply:
yes